docs/source/developers/continuous_integration/docker.rst

```diff
@@ -156,6 +156,17 @@ The following example starts an interactive ``bash`` session in the container
 
     archery docker run ubuntu-cpp bash
 
+**Build the image with increased debugging output:**
+
+To enable additional logging output for debugging, pass the ``--debug`` flag
+to ``archery``.
+
+.. code:: bash
+
+    archery --debug docker run ubuntu-cpp
+
+which translates to passing ``--progress=plain`` to docker(-compose) build command.
```

Comment:
Not only, it also outputs the actual docker commands being executed (and perhaps other Archery logging).

Reply:
Would you be OK with it if I just add something like "which, among other things, ..." ? 

I don't know how specific / complete we want to be here (I also don't really know what the debug enabled exactly), but I mostly wanted to not that specifically for the `build` part it enabled the progress=plain (because that was the flag I wanted to enable based on googling)

Reply:
From a quick search in the archery code, I think it essentially does right now:

Generally (not docker specific):

- set `logger.setLevel(logging.DEBUG)`

And then for the docker subcommand:

- log docker version
- add `--progress=plain` to docker build and docker-compose build

Reply:
Rephrased it to be more complete.